pkg/testing/fixture_install.go

```diff
@@ -601,8 +609,9 @@ func (f *Fixture) installRpm(ctx context.Context, installOpts *InstallOpts, shou
 }
 
 type UninstallOpts struct {
-	Force          bool // --force
-	UninstallToken string
+	Force          bool   // --force
+	UninstallToken string // --uninstall-token
+	SkipFleetAudit bool   // --skip-fleet-audit
```

Comment:
this param `--skip-fleet-audit` is not mentioned int he PR description. Is this a left-over? if not, please do update the description to capture it